routers/repo/editor_test.go

```diff
@@ -15,14 +15,17 @@ func TestCleanUploadName(t *testing.T) {
 	models.PrepareTestEnv(t)
 
 	var kases = map[string]string{
-		".git/refs/master": "git/refs/master",
+		".git/refs/master": "",
 		"/root/abc":        "root/abc",
 		"./../../abc":      "abc",
-		"a/../.git":        "a/.git",
-		"a/../../../abc":   "a/abc",
+		"a/../.git":        "",
+		"a/../../../abc":   "abc",
 		"../../../acd":     "acd",
-		"../../.git/abc":   "git/abc",
-		"..\\..\\.git/abc": "git/abc",
+		"../../.git/abc":   "",
+		"..\\..\\.git/abc": "..\\..\\.git/abc",
+		"..\\../.git/abc":  "",
+		"..\\../.git":      "",
+		"abc/../def":       "def",
```

Comment:
Can you add a test for files that have periods at the start (such as `.drone.yml`)?

Reply:
Done